OpenEdXMobile/src/main/java/org/edx/mobile/util/TextUtils.java

```diff
@@ -244,4 +249,31 @@ public static SpannableString underline(Context context, @StringRes int resId) {
         content.setSpan(new UnderlineSpan(), 0, content.length(), 0);
         return content;
     }
+
+    public static SpannableString setIconifiedText(Context context, String text, @DrawableRes int iconResId, View.OnClickListener iconClickListener) {
```

Comment:
deserve doc string, including that method used to add icon at the end to the text.